src/utils.py

```diff
@@ -16,4 +19,15 @@ def epoch_time(start_time, end_time):
     elapsed_time = end_time - start_time
     elapsed_mins = int(elapsed_time / 60)
     elapsed_secs = int(elapsed_time - (elapsed_mins * 60))
-    return elapsed_mins, elapsed_secs
\ No newline at end of file
+    return elapsed_mins, elapsed_secs
+
+
+def count_parameters(model):
```

Comment:
This looks good, but would you mind adding a docstring just describing what it will do and return parameter typedefinitions?

Reply:
Good catch! Added now.